core/codegen/src/lib.rs

```diff
@@ -774,6 +774,33 @@ pub fn derive_from_form(input: TokenStream) -> TokenStream {
     emit!(derive::from_form::derive_from_form(input))
 }
 
+/// Derive for the [`FromParam`] trait.
+///
+/// The [`FromParam`] derive can be applied to enums with nullary
```

Comment:
We need some description of what the macro produces. Is it a parser that stringifies each variant and then case-sensitively matches the stringified variant to the respective enum variant? If so, we should say that.